pkgs/containers/nvidia-ctk.nix

```diff
@@ -0,0 +1,38 @@
+{ fetchFromGitLab, buildGoModule, fetchpatch2 }:
+
+buildGoModule rec {
+  pname = "nvidia-ctk";
+  version = "unstable-${builtins.substring 0 7 src.rev}";
+
+  src = fetchFromGitLab {
```

Comment:
This pins to a newer rev to get features for a soon-to-be released v1.15 of nvidia-container-toolkit. We should have all the features we need once rc2 of v1.15 is released, which should be any day. But definitely will be pinned to a release version once 1.15 is released.

Reply:
Can you add a comment in the code itself saying this?

Reply:
added!